packages/calico/README.md

```diff
@@ -0,0 +1,323 @@
+  "container": {
+    "type": "MESOS",
+    "volumes": [],
+    "docker": {
+      "image": "mesosphere/id-server:2.1.0"
+    },
+    "portMappings": []
+  },
+  "cpus": 0.1,
+  "mem": 128,
+  "requirePorts": false,
+  "networks": [
+    {
+      "mode": "container",
+      "name": "calico",
+      "labels": {
+        "role": "client"
+      }
+    }
+  ],
+  "healthChecks": [],
+  "fetch": [],
+  "constraints": []
+}
+```
+
+### default profile
+
+Calico Profile groups endpoints which inherit labels defined in the profile, for example, each namespace has one corresponding profile to granting labels to Pods in the namespace. Calico profile supports policy rules for traffic control but is deprecated in favor of much more flexible NetworkPolicy and GlobalNetworkPolicy resources.
+
+In our case, all Calico networking containers will be assigned with a default profile with the same name as CNI network, `calico` by default, and this profile allows **only** requests from one Calico container network to another one,  which means L4LB and L7 proxy requests in which the source IP address is NATed to that of tunnel interfaces generated by Calico, and finally will be dropped. This profile can found in the following YAML definition.
+```yaml
+apiVersion: projectcalico.org/v3
+kind: Profile
+metadata:
+  creationTimestamp: 2019-12-23T04:16:55Z
+  name: calico
+  resourceVersion: "75"
+  uid: 0e105ecb-253b-11ea-9e52-065b6833052c
+spec:
+  egress:
+  - action: Allow
+    destination: {}
+    source: {}
+  ingress:
+  - action: Allow
+    destination: {}
+    source:
+      selector: has(calico)
+  labelsToApply:
+    calico: ""
+```
+To resolve this problem, calico profile `calico` is initialized by default by `dcos-calico-felix`, and allows all traffic into and out of Calico networking containers, and `calico` is the only profile supported for now and shared across all Calico networking containers.
+For a more detailed description of the Calico profile, please read [here](https://docs.projectcalico.org/v3.8/reference/resources/profile).
+
+## Example
+
+### Calico networking containers
+
+To use Calico networking containers, you only have to specify the network name as `calico`, and the following example shows the Marathon application definition of Calico networking containers supported by Mesos UCR.
+a Marathon application with Calcico networking supported by Mesos UCR:
+```
+{
+  "id": "/calico-ucr",
+  "instances": 1,
+  "container": {
+    "type": "MESOS",
+    "volumes": [],
+    "docker": {
+      "image": "mesosphere/id-server:2.1.0"
+    },
+    "portMappings": []
+  },
+  "cpus": 0.1,
+  "mem": 128,
+  "requirePorts": false,
+  "networks": [
+    {
+      "mode": "container",
+      "name": "calico"
+    }
+  ],
+  "healthChecks": [],
+  "fetch": [],
+  "constraints": []
+}
+```
+
+TODO: Or you could create a Calico networking Service through DC/OS web interface, tracked in [DCOS-62535](https://jira.mesosphere.com/browse/DCOS-62535)
```

Comment:
Total nit-picking, but since this documentation is public-facing, I would recommend avoiding "TODO" and doing something like:

> Note: Support for Calico networking Service configuration though the DC/OS web interface is coming soon. Refer to [DCOS-62535](https://jira.mesosphere.com/browse/DCOS-62535) for more details.